data_validation/src/spark_rapids_validation_tool/csp/dataproc.py

```diff
@@ -58,21 +58,21 @@ def get_info(self):
 
     def get_nodes(self, node='all'):
         """Get cluster node address."""
-        # node format: <all|master|workers|workers-n>
+        # node format: <all|primary|executors|executors-n>
```

Comment:
Would be good to replace `primary` with `driver` as done in other files.

Reply:
Fixed